tools/tool-base/src/main/java/io/spine/code/fs/DirectoryReference.java

```diff
@@ -54,10 +54,17 @@ public static DirectoryReference of(String value) {
         return new DirectoryReference(value);
     }
 
+    /**
+     * Obtains the reference to the current directory.
+     */
+    public static DirectoryReference currentDir() {
```

Comment:
Similar to my comment below, I would expect that the reference to the current dir would be either `"."` or `"." + File.separatorChar`.